### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by enis-memic**
In this following branch I did the following:

- Implemented desktop version with media queries
- Fixed html file of the page so I can adjust the media queries and style to it.
- Fixed all the styling needed for the whole page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,6 +22,13 @@ <h2>My Logo</h2>
...
       </nav>
+      <nav class="navigation-links">
+        <ul class="nav-links">
+          <li><a href="#" class="hover-menu">Portfolio</a></li>
+          <li><a href="#" class="hover-menu">About</a></li>
+          <li><a href="#" class="hover-menu">Contact</a></li>
+        </ul>
+      </nav>
     </header>
     <section class="intro-section">
```
Comment (wayungi) [inline index.html:33]: - [ ] Please add the background images as indicated on the Figma template.

_**Figma**_
figma-bg

_**Your design**_
figma-bg-2

File: index.html [hunk 2]
```diff
@@ -308,6 +315,7 @@ <h4 class="tabs-header">Skills</h4>
         </ul>
       </div>
     </section>
```
Comment (wayungi) [inline index.html:317]: - [ ] The ordering of your Projects is not consistent with the Desktop version ordering. Please use the [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) order attribute in your desktop media query to achieve this feature. 

_**figma**_
ordering-1

_**Your design**_
ordering-2

File: index.html [hunk 3]
```diff
@@ -340,5 +348,6 @@ <h2 class="contact-title">Contact me</h2>
         class="btn btn-contact submit"></input>
       </form>
```
Comment (wayungi) [inline index.html:349]: - [ ] Please make sure your contact form background matches the Figma design.

**_Figma_**
form-1


_**Your design**_
form-2

File: index.html (hunk 2 above)
Comment (wayungi) [reply index.html:317]: - [ ] Please note that in the section on **About Myself**, The left side is 40% of the page width and the right side is 60% of the page width. In your implementation, they each take 50% of the page width. Please change this. 🧑‍💼 

_**figma**_
about-1


_**Your design**_
about-2

Comment (wayungi) [reply index.html:317]: - [ ]  Please note that **Languages, Frameworks & skills** on the right side of the About Myself section are drop-down menus. These have not been implemented in your design. Kindly stick to the figma template 🧑‍🏭

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @enis-memic , 

You have done a very good job coming far! 👏🏾 
There are some issues that you still need to work on 🧑‍🏭  to go to the next project but you are almost there! 
image

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- [ ] Good commit messages ✔️ 
- [ ] Descriptive PR title ✔️ 
- [ ] Correct GitHub flow ✔️ 
- [ ] Descritive PR body ✔️ 
- [ ] Linter checks all passing ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (enis-memic) [reply index.html:33]: @wayungi  I do have background uploaded on my side and I have tried it on some other browsers and computers and it is always showing the background image.

Screenshot from 2023-01-18 15-30-32

File: index.html (hunk 2 above)
Comment (enis-memic) [reply index.html:317]: @wayungi can you clarify what do you mean they have not been implemented, it looks exactly the same like that in Figma template

File: index.html (hunk 3 above)
Comment (enis-memic) [reply index.html:349]: @wayungi  here is the same like for the header background. On my web page I can see the background and also on other browsers. As you can see in the photo below


Screenshot from 2023-01-18 15-34-40

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @enis-memic  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] Check the comments under the review.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**